obspy/taup/tau_model.py

```diff
@@ -25,26 +25,14 @@ class TauModel(object):
     """
     Provides storage of all the TauBranches comprising a model.
     """
-    # The following really need to be class attributes. For some reason.
-
-    # Depth for which tau model as constructed.
-    source_depth = 0.0
-    radiusOfEarth = 6371.0
-    # Branch with the source at its top.
-    sourceBranch = 0
-    # Depths that should not have reflections or phase conversions. For
-    # instance, if the source is not at a branch boundary then
-    # noDisconDepths contains source depth and reflections and phase
-    # conversions are not allowed at this branch boundary. If the source
-    # happens to fall on a real discontinuity then then it is not
-    # included.
-    noDisconDepths = []
-
-    def __init__(self, sMod, spherical=True, debug=False, skip_calc=False):
+    def __init__(self, sMod, radius_of_planet, is_spherical=True, debug=False,
+                 skip_calc=False):
         self.debug = debug
-        self.radiusOfEarth = 6371.0
-        # True if this is a spherical slowness model. False if flat.
-        self.spherical = spherical
+        # Depth for which tau model as constructed.
+        self.source_depth = 0.0
+        self.radius_of_planet = radius_of_planet
+        # True if this is a is_spherical slowness model. False if flat.
```

Comment:
Not sure if the `is_` is necessary in the variable, but it's definitely not needed in this comment.